From 8d5f37df2c59f9ae7f461908fe57c0ff58c180a6 Mon Sep 17 00:00:00 2001
From: Piotr Ziecik <kosmo@semihalf.com>
Date: Tue, 25 Mar 2014 10:55:56 +0100
Subject: [PATCH 1507/1825] alp/axp: Enable FreeRTOS IPC when Dragonite
 support is compiled in.

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 3df56be5a273b268b0d6a86f13e18da42fb97f40

	The FreeRTOS IPC can be now used to communicate with Dragonite.

Change-Id: I7615c754baa12984aab88845a61841c59f21526f
Signed-off-by: Piotr Ziecik <kosmo@semihalf.com>
Signed-off-by: Victor Gu <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/6667
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
Tested-by: Yehuda Yitschak <yehuday@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/plat-armada/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-armada/Kconfig b/arch/arm/plat-armada/Kconfig
index a06136d..c3f2cd2 100644
--- a/arch/arm/plat-armada/Kconfig
+++ b/arch/arm/plat-armada/Kconfig
@@ -306,7 +306,7 @@ config MV_IPC_NET
 
 config MV_IPC_FREERTOS_DRIVER
         bool "Enable freeRTOS IPC support"
-        depends on   MV_AMP_ENABLE && !MV_IPC_LINUX_AMP_DRIVER
+        depends on (MV_AMP_ENABLE || MV_INCLUDE_DRAGONITE) && !MV_IPC_LINUX_AMP_DRIVER
 	default n
 
 config MV_DRAM_BASE
-- 
1.7.5.4

